builder/src/section/choice-panel.tsx

```diff
@@ -522,6 +586,20 @@ export const ChoicePanelFactory =
             />
             {generateChoiceDividerPositionInputElements(index)}
             {generateChoiceNoneChoiceInputElements(index)}
+            <button hidden={currentNumberOfChoices < 2 || !showAllFields}
+              id={"delete-button-" + index}
+              class="btn-small btn-danger"
+              onClick={(e) => { removeChoice(e, index); } }
+            >
```

Comment:
Placement of remove button seems non-intuitive, because it is on the same line as following option. It would be better if it was on the first line of the choice it is actually deleting.